### PR TITLE
Sandoche/fse 665 governance e2e

### PR DESCRIPTION
**Pull request by sandoche**
- Adding e2e test to governance app
- Creating a shared config
- Adding playwright to all the apps
- Adding GH Action
- Adding dAppWright to interact with metamask

## Future changes
- (fse-675) the code will be refactored so the utilities can be re-used and the seed will be updated in .env
- (fse-676) the github flow will be refactored to avoid having to build 3 times in different flows

### CODE REVIEW

**Comment by sandoche** [comment]
Fixes to try:
- Update the ports for each app separately
- Increase the time
- Try to display logs to debug

File: apps/staking/playwright.config.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+/**
+ * Read environment variables from file.
+ * https://github.com/motdotla/dotenv
+ */
+// require('dotenv').config();
+
+/**
+ * See https://playwright.dev/docs/test-configuration.
+ */
+export default defineConfig({
```
[the remark this answers is not in the document]
Comment (sandoche) [reply apps/staking/playwright.config.ts:18]: Will be fixed in fse-675

File: apps/vesting/playwright.config.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+/**
+ * Read environment variables from file.
+ * https://github.com/motdotla/dotenv
+ */
+// require('dotenv').config();
+
+/**
+ * See https://playwright.dev/docs/test-configuration.
+ */
+export default defineConfig({
```
[the remark this answers is not in the document]
Comment (sandoche) [reply apps/vesting/playwright.config.ts:18]: Will be fixed in fse-675

File: apps/governance/playwright.config.ts [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+/**
+ * Read environment variables from file.
+ * https://github.com/motdotla/dotenv
+ */
+require("dotenv").config();
+
+/**
+ * See https://playwright.dev/docs/test-configuration.
+ */
+export default defineConfig({
```
[the remark this answers is not in the document]
Comment (sandoche) [reply apps/governance/playwright.config.ts:18]: Will be fixed in fse-675

**Review by miliPaoletti — APPROVED** [review]

**Pull request merged**